Gruntfile.js

```diff
@@ -214,9 +214,19 @@ const configureGrunt = function (grunt) {
                 command: function () {
                     var upstream = grunt.option('upstream') || process.env.GHOST_UPSTREAM || 'upstream';
                     grunt.log.writeln('Pulling down the latest master from ' + upstream);
-                    return 'git checkout master; git pull ' + upstream + ' master; ' +
-                        'yarn; git submodule foreach "git checkout master && git pull ' +
-                        upstream + ' master"';
+                    return `
```

Comment:
I'm wondering if it makes sense to have smth like `scripts/master.sh` and just call out to that from here :man_shrugging:

Reply:
The script is small and useful to see in the Gruntfile. I would suggest keeping it this way unless it needs to be called from multiple places.

Reply:
Makes sense to me :relaxed: